example/README.md

```diff
@@ -0,0 +1,63 @@
+# Demo Instructions
+
+This demo will show you how to subscribe, confirm a subscription and publish a message to a topic
+
+As the sample code includes the private key verbatim in the source, it should be treated carefully, and not checked into version control!
+
+
+## Setting up IAM Policy
+
+1. Select `Services` link (on the top left of the page) and them type `IAM` in the search line
+1. Select `IAM Manage User Access and Encryption Keys` item
+1. Select `Policies` item from the menu on the left
+1. Press `Create Policy` button
+1. Press `Select` for `Policy Generator`
+1. On the `Edit Permissions` page do the following
+    1. Set `Effect` to `Allow`
+    1. Set `AWS Service` to `Amazon SNS`
+    1. Set `Actions` to `All Actions`
+    1. Leave `Amazon Resource Name (ARN)` blank
```

Comment:
You can't actually leave this blank. I had to fix the same here https://github.com/electricimp/AWSSQS/tree/develop/example

Reply:
Setting up a topic first worked for me.

Reply:
exactly it should be reordered